.gitattributes

```diff
@@ -11,7 +11,9 @@
 /Resources/Private/package.json export-ignore
 /Resources/Private/stylelint.config.js export-ignore
 /Tests/ export-ignore
+/badge.svg export-ignore
 /codeception.yml export-ignore
+/clover.xml export-ignore
```

Comment:
I'd prefer to git-ignore `clover.xml` instead and keep it out of the repo. (Otherwise, we'd have to include the coverage changes in each PR.)